packages/apsara-ui/src/Combobox/Combobox.tsx

```diff
@@ -82,7 +83,7 @@ const Combobox = ({
             onDeselect={onValueDeselect}
             options={options}
             filterOption={true}
-            optionFilterProp="label"
+            optionFilterProp={optionFilterProp}
```

Comment:
```suggestion
            optionFilterProp={optionFilterProp || 'value'}
```

we can do the default thing here